understanding/21/pointer-gestures.html

```diff
@@ -3,23 +3,39 @@
+
+			<p>A <strong>path-based gesture</strong> involves an interaction where the user engages a pointer (starting point), carries out a movement that goes through at least one mid-point before disengaging the pointer (end point). The mid-point defines the path as a gesture.</p>
+
+			<figure>
+				<img src="img/path-based-gesture-1.png" width="400" alt="Hand showing a starting touch, 1. Moving through a second point, 2. Going to one of several points,3.">
+				<figcaption>A path-based gesture involves starting a pointer movement that goes through at least one mid-point before the end-point. The end-point may be a continuation, or allow for various movements.</figcaption>
+			</figure>
+
+			<p>Examples of path-based gestures include swiping, some sliders and carousels, and other gestures which trace a prescribed path such as the drawing a specific shape. Such paths may be drawn with a finger or stylus on a touchscreen, graphics tablet, or trackpad, or with a mouse, joystick, or similar pointer device.</p>
+
+			<p class="note">Any path-based movement of a pointer could be difficult or impossible to use for someone who cannot perform precise movements, therefore alternative forms of interaction are always recommended. Path-based gestures are used by the success criteria as it may be difficult or impossible to provide an alternative for free-form paths.</p>
+
+			<p>Examples of <strong>multipoint</strong> gestures include a two-finger pinch zoom, a split tap where one finger rests on the screen and a second finger taps, or a two- or three-finger tap or swipe. A user may find it difficult or impossible to accomplish these if they type and point with a single finger or stick, in addition to any of the causes listed above.</p>
+
+			<p>Authors must ensure that their content can be operated without multipoint or path-based gestures. Multipoint or path-based gestures can be used so long as the functionality can also be operated by another method, such as a tap, click, double tap, double click, long press, and click &amp; hold. </p>
+
+			<p>This Success Criterion applies to author-created gestures, as opposed to gestures defined on the level of operating system or user agent. Examples for gestures provided at the operating system level would be swiping down to see system notifications, and gestures for built-in assistive technologies (AT) to focus or activate content, or to call up AT menus. Examples of user-agent-implemented gestures would be horizontal swiping implemented by browsers for navigating within the page history, or vertical swiping to scroll page content.</p>
+
 			<p>While some operating systems may provide ways to define "macros" to replace complex gestures, content authors cannot rely on such a capability because it is not pervasive on all touch-enabled platforms. Moreover, this may work for standard gestures that a user can predefine, but may not work for other author-defined gestures.</p>
```

Comment:
Maybe "supported" or "available" instead of "pervasive"?

Reply:
Still applies. Just trying to simplify language, even if a little amount.

Reply:
Which is odd, as the word 'pervasive' isn't in the file anymore!?